src/Simplex/Chat/Store.hs

```diff
@@ -380,81 +394,99 @@ getGroup st User {userId, userContactId} localDisplayName =
               c.connection_id, c.agent_conn_id, c.conn_level, c.via_contact,
               c.conn_status, c.conn_type, c.contact_id, c.group_member_id, c.created_at
             FROM group_members m
-            JOIN groups g ON g.group_id = m.group_id
             JOIN contact_profiles p ON p.contact_profile_id = m.contact_profile_id
-            JOIN connections c ON c.group_member_id = m.group_member_id
-            WHERE g.group_id = ?
-            ORDER BY c.connection_id DESC
-            LIMIT 1
+            LEFT JOIN connections c ON c.connection_id = (
+              SELECT max(cc.connection_id)
+              FROM connections cc
+              where cc.group_member_id = c.group_member_id
```

Comment:
shouldn't it be "where cc.group_member_id = **m**.group_member_id"? or it doesn't matter..? - got a little confused

Reply:
at least we were definitely verifying the "m"

Reply:
so it was wrong to remove extra equality - it was `c.group_member_id = m.group_member_id` I think

how it even works?